packages/common/src/ceramic-api.ts

```diff
@@ -59,25 +59,25 @@ export interface CeramicApi extends CeramicSigner {
     // loggerProvider: LoggerProvider; // TODO uncomment once logger is available on http-client
 
     /**
-     * Register Doctype handler
-     * @param doctypeHandler - DoctypeHandler instance
+     * Register Stream handler
+     * @param doctypeHandler - StreamHandler instance
      */
-    addDoctypeHandler<T extends Doctype>(doctypeHandler: DoctypeHandler<T>): void;
+    addStreamHandler<T extends Stream>(doctypeHandler: StreamHandler<T>): void;
```

Comment:
Most places I felt made sense to rename `Doctype` to `Stream`.  This is the one place I'm not 100% on.  I could see calling the class `StreamTypeHandler` and this method `addStreamTypeHandler`, if others prefer that.

Reply:
Don't have a strong preference. StreamHandler should be fine.